<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1651</width>
    <height>1149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="CentralView">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="main_window_layout">
      <item>
       <widget class="QGroupBox" name="left_panel">
        <property name="minimumSize">
         <size>
          <width>370</width>
          <height>500</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Immediate Results</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QScrollArea" name="left_scroll">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="left_scroll_results">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>354</width>
              <height>1093</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <layout class="QVBoxLayout" name="left_scroll_results_vbox_layout"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="mid_panel">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>500</height>
         </size>
        </property>
        <property name="title">
         <string>Current Selection</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QScrollArea" name="main_image_scroll">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="main_scoll_vbox_layout">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>869</width>
              <height>1093</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QLabel" name="main_image_label">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="right_panel">
        <property name="minimumSize">
         <size>
          <width>370</width>
          <height>500</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Details</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="3">
          <widget class="QPushButton" name="new_btn">
           <property name="minimumSize">
            <size>
             <width>31</width>
             <height>31</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QPushButton" name="delete_btn">
           <property name="minimumSize">
            <size>
             <width>31</width>
             <height>31</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="fav_label">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Preset</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QPushButton" name="save_btn">
           <property name="minimumSize">
            <size>
             <width>31</width>
             <height>31</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QLineEdit" name="custom_line_edit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QComboBox" name="fav_pips_combo_box">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="custom_label">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Custom</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" rowspan="2" colspan="6">
          <widget class="QGroupBox" name="pip_collapsable">
           <property name="minimumSize">
            <size>
             <width>350</width>
             <height>300</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>300</height>
            </size>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="pip_collapse_button">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Pipeline</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_collapse_pipe">
                <property name="maximumSize">
                 <size>
                  <width>25</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>down.png</normaloff>
                  <normalon>up.png</normalon>down.png</iconset>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QScrollArea" name="pip_scroll">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="pip_scroll_vbox_layout">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>338</width>
                 <height>244</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <layout class="QVBoxLayout" name="pip_widget_vbox_layout">
                  <property name="spacing">
                   <number>2</number>
                  </property>
                  <item>
                   <widget class="QWidget" name="widget" native="true">
                    <property name="font">
                     <font>
                      <underline>false</underline>
                     </font>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout">
                     <item>
                      <widget class="QLabel" name="current_pip_label">
                       <property name="font">
                        <font>
                         <pointsize>10</pointsize>
                        </font>
                       </property>
                       <property name="text">
                        <string>[Current_Pip_Name]</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="add_btn">
                       <property name="minimumSize">
                        <size>
                         <width>31</width>
                         <height>31</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>31</width>
                         <height>31</height>
                        </size>
                       </property>
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="Line" name="line">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="4" column="0" colspan="6">
          <widget class="QGroupBox" name="settings_collapsable">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>350</width>
             <height>400</height>
            </size>
           </property>
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="title">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="settings_collapse_button">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Settings</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_collapse_settings">
                <property name="minimumSize">
                 <size>
                  <width>1</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>25</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string notr="true"/>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>down.png</normaloff>
                  <normalon>up.png</normalon>down.png</iconset>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="default">
                 <bool>false</bool>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QScrollArea" name="setting_scroll">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="settings_scroll_vbox_layout">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>338</width>
                 <height>653</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <property name="topMargin">
                 <number>0</number>
                </property>
                <item>
                 <layout class="QVBoxLayout" name="setting_widget_vbox_layout"/>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
        <zorder>custom_line_edit</zorder>
        <zorder>custom_label</zorder>
        <zorder>fav_label</zorder>
        <zorder>fav_pips_combo_box</zorder>
        <zorder>settings_collapsable</zorder>
        <zorder>pip_collapsable</zorder>
        <zorder>verticalSpacer</zorder>
        <zorder>new_btn</zorder>
        <zorder>save_btn</zorder>
        <zorder>delete_btn</zorder>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="PipelineMenu">
   <property name="text">
    <string>Pipeline</string>
   </property>
   <property name="toolTip">
    <string>Menuentrys for saving and loading pipelines</string>
   </property>
  </action>
  <action name="actionImages">
   <property name="text">
    <string>Images</string>
   </property>
   <property name="toolTip">
    <string>Menu for saving and loading images</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
   <property name="toolTip">
    <string>Load a presaved pipeline in json format</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save the pipeline settings in a json format. You can load your specified Pipeline in later program calls to process the same settings on other images.</string>
   </property>
  </action>
  <action name="actionLoad_2">
   <property name="text">
    <string>Load</string>
   </property>
   <property name="toolTip">
    <string>Load a presaved Pipeline in json format.</string>
   </property>
  </action>
  <action name="actionSave_2">
   <property name="text">
    <string>Specify Output location</string>
   </property>
   <property name="toolTip">
    <string>Specify the default output location in yout file system. Your processed images and graphs will be stored there after processing.</string>
   </property>
  </action>
  <action name="actionLoad_3">
   <property name="text">
    <string>Load Image</string>
   </property>
   <property name="toolTip">
    <string>Load a single image file from the file system.</string>
   </property>
  </action>
  <action name="actionLoad_Image_Folder">
   <property name="text">
    <string>Load Image Folder</string>
   </property>
   <property name="toolTip">
    <string>Load a complete folder of images. To display immediate results, the program will use the first image of the folder as default.</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
